### PR TITLE
update number/tolocalestring

### PR DESCRIPTION
**Pull request by jasonren0403**
Remove `{{page}}` occurence and update with newest EN document.

Related issue: #3893

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/number/tolocalestring/index.md [hunk 1]
```diff
@@ -3,76 +3,94 @@ title: Number.prototype.toLocaleString()
...
-查阅[浏览器兼容性](#浏览器兼容性)部分，了解哪些浏览器支持 `locales` 和 `options` 参数，通过[示例：检查 `locales` 和 `options` 参数的支持](#Checking_for_support_for_locales_and_options_arguments)了解特征检测。
+`locales` 和 `options` 参数让应用程序可以指定要进行格式转换的语言，并且定制函数的行为。
 
-> **备注：** ECMAScript 国际化 API，在 Firefox 29 中得以实施，增加了 `locales` 参数的 [`Number.toLocaleString`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) 方法。如果参数为 `undefined`，此方法返回本地操作系统指定的位数，而 Firefox 的早期版本中返回[阿拉伯语](https://en.wikipedia.org/wiki/Arabic_numerals)数字。这一变化已被报告为向后影响的兼容性问题并可能会被尽快修复。({{ bug(999003) }})
+在使用具有 [`Intl.NumberFormat` API](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) 支持的实现时，这些参数与 [`Intl.NumberFormat()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat) 构造器的参数相同。不支持 `Intl.NumberFormat` 时，会忽略参数，使得使用的区域设置和返回的字符串格式完全由实现决定。
 
-{{page('/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat', '参数')}}
+- `locales` {{optional_inline}}
+
+  - : 缩写语言代码（BCP 47 language tag，例如: cmn-Hans-CN）的字符串或者这些字符串组成的数组。与 `Intl.NumberFormat()` 构造器的 [`locales`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#locales) 参数相同。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/number/tolocalestring/index.md:28]:
```suggestion
  - : 缩写语言代码（BCP 47 language tag，例如 cmn-Hans-CN）的字符串或者这些字符串组成的数组。与 `Intl.NumberFormat()` 构造器的 [`locales`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#locales) 参数相同。
```

File: files/zh-cn/web/javascript/reference/global_objects/number/tolocalestring/index.md [hunk 2]
```diff
@@ -3,76 +3,94 @@ title: Number.prototype.toLocaleString()
...
-查阅[浏览器兼容性](#浏览器兼容性)部分，了解哪些浏览器支持 `locales` 和 `options` 参数，通过[示例：检查 `locales` 和 `options` 参数的支持](#Checking_for_support_for_locales_and_options_arguments)了解特征检测。
+`locales` 和 `options` 参数让应用程序可以指定要进行格式转换的语言，并且定制函数的行为。
 
-> **备注：** ECMAScript 国际化 API，在 Firefox 29 中得以实施，增加了 `locales` 参数的 [`Number.toLocaleString`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) 方法。如果参数为 `undefined`，此方法返回本地操作系统指定的位数，而 Firefox 的早期版本中返回[阿拉伯语](https://en.wikipedia.org/wiki/Arabic_numerals)数字。这一变化已被报告为向后影响的兼容性问题并可能会被尽快修复。({{ bug(999003) }})
+在使用具有 [`Intl.NumberFormat` API](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) 支持的实现时，这些参数与 [`Intl.NumberFormat()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat) 构造器的参数相同。不支持 `Intl.NumberFormat` 时，会忽略参数，使得使用的区域设置和返回的字符串格式完全由实现决定。
 
-{{page('/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat', '参数')}}
+- `locales` {{optional_inline}}
+
+  - : 缩写语言代码（BCP 47 language tag，例如 cmn-Hans-CN）的字符串或者这些字符串组成的数组。与 `Intl.NumberFormat()` 构造器的 [`locales`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#locales) 参数相同。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/tolocalestring/index.md:28]:
```suggestion
  - : 缩写语言代码（BCP 47 language tag）的字符串或者这些字符串组成的数组。与 `Intl.NumberFormat()` 构造函数的 [`locales`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#locales) 参数相同。
```

File: files/zh-cn/web/javascript/reference/global_objects/number/tolocalestring/index.md [hunk 3]
```diff
@@ -3,76 +3,94 @@ title: Number.prototype.toLocaleString()
...
+
+    在无 `Intl.NumberFormat` 支持的实现中，往往使用主机的区域设置，这个参数是忽略的。
+
+- `options` {{optional_inline}}
+
+  - : 调整输出格式的对象。与 `Intl.NumberFormat()` 构造器的 [`options`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#options) 参数相同。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/tolocalestring/index.md:34]:
```suggestion
  - : 调整输出格式的对象。与 `Intl.NumberFormat()` 构造函数的 [`options`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#options) 参数相同。
```

File: files/zh-cn/web/javascript/reference/global_objects/number/tolocalestring/index.md [hunk 4]
```diff
@@ -3,76 +3,94 @@ title: Number.prototype.toLocaleString()
...
+
+    在无 `Intl.NumberFormat` 支持的实现中，这个参数是忽略的。
+
+请查阅 [`Intl.NumberFormat()` 构造器](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat)以了解这些参数的详细信息，以及如何使用它们。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/tolocalestring/index.md:38]:
```suggestion
请查阅 [`Intl.NumberFormat()` 构造函数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat)以了解这些参数的详细信息，以及如何使用它们。
```

File: files/zh-cn/web/javascript/reference/global_objects/number/tolocalestring/index.md [hunk 5]
```diff
@@ -3,76 +3,94 @@ title: Number.prototype.toLocaleString()
...
+    return e.name === 'RangeError';
   }
   return false;
 }
 ```
 
-早于 ES5.1 的实现中，如果带参数调用 `toLocaleString` 并不会抛出范围异常。
-
-在所有宿主环境下，包括那些支持比 ed 5.1 还早的 ECMA-262 的环境，都能有效检测的方法是直接检测 ECMA-402 中的其它特性，它指定 `Number.prototype.toLocaleString` 需要支持地区选项：
+早于 ES5.1 的实现中，携带非法参数调用 `toLocaleString` 不需要抛出范围异常。在所有宿主环境下，包括那些支持比 ed 5.1 还早的 ECMA-262 的环境，都能有效检测的方法是直接检测 ECMA-402 中的其它特性，它指定 `Number.prototype.toLocaleString` 需要支持地区选项：
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/tolocalestring/index.md:78]: 全篇统一使用“主机”：

```suggestion
早于 ES5.1 的实现中，携带非法参数调用 `toLocaleString` 不需要抛出范围异常。在所有主机环境下，包括那些支持比 ed 5.1 还早的 ECMA-262 的环境，都能有效检测的方法是直接检测 ECMA-402 中的其它特性，它指定 `Number.prototype.toLocaleString` 需要支持地区选项：
```

**Review by yin1999 — APPROVED** [review]
Thanks a lot. We are on the way to no `{{page}}` macros.

**Pull request merged**